src/index.ts

```diff
@@ -1,26 +1,52 @@
 import { SNS } from 'aws-sdk'
 import * as _find from 'lodash.find'
 import * as request from 'request-promise'
-
 import epsagon from 'epsagon'
 
-import * as config from './config'
+import { EPSAGON } from './config'
 
 epsagon.init({
-  appName: config.EPSAGON_APP_NAME,
+  appName: EPSAGON.APP_NAME,
   metadataOnly: false,
-  token: config.EPSAGON_ACCOUNT_TOKEN,
+  token: EPSAGON.ACCOUNT_TOKEN,
 })
 
-import db from './database'
+import db, { upsertCars } from './database'
```

Comment:
Remove the default on the db in database.ts and then make this line 
```suggestion
import { db, upsertCars } from './database'
```